### ISSUE TITLE
backprop through loss

### ISSUE THREAD

**Issue by pryadchenko** [sgrvinod/a-PyTorch-Tutorial-to-Object-Detection#37]
Hello. My apologies for such a (probably) stupid question, but I can not understand how does backpropagation works through the label matching operation via Jaccard index comparison. Can you, please, help me to find an explanation?
Thanks in advance.
